docs/api.rst

```diff
@@ -519,6 +519,46 @@ Deleting premium devices
    :statuscode 403: Logged in user does not have premium.
    :statuscode 409: The external premium service could not be reached or returned unexpected response.
 
+Querying premium feature flags
```

Comment:
Can we generalize this? Not feature flags but just `capabilities` for the given tier. Rephrase the entire docs around that. Because we may indeed need to return numbers/limits to the frontend or some other consumer.